hbase/hbase/stackable/bin/hbase-entrypoint.sh

```diff
@@ -72,6 +75,16 @@ if [ -f /stackable/kerberos/krb5.conf ]; then
   sed -i -e s/\$\{env\.KERBEROS_REALM\}/"${KERBEROS_REALM}"/g /stackable/conf/hdfs-site.xml
 fi
 
+# Service endpoints
+HBASE_SERVICE_PORT=$(cat /stackable/listener/default-address/ports/"${HBASE_PORT_NAME}")
+HBASE_INFO_PORT=$(cat /stackable/listener/default-address/ports/"${HBASE_UI_PORT_NAME}")
+HBASE_LISTENER_ENDPOINT="$HBASE_SERVICE_HOST:$HBASE_INFO_PORT"
+
+sed -i -e s/\$\{HBASE_SERVICE_HOST\}/"${HBASE_SERVICE_HOST}"/g /stackable/conf/hbase-site.xml
+sed -i -e s/\$\{HBASE_SERVICE_PORT\}/"${HBASE_SERVICE_PORT}"/g /stackable/conf/hbase-site.xml
+sed -i -e s/\$\{HBASE_LISTENER_ENDPOINT\}/"${HBASE_LISTENER_ENDPOINT}"/g /stackable/conf/hbase-site.xml
+sed -i -e s/\$\{HBASE_INFO_PORT\}/"${HBASE_INFO_PORT}"/g /stackable/conf/hbase-site.xml
```

Comment:
Any reason for this to not be using https://github.com/stackabletech/config-utils instead of sed? AFAIU we already pull it in via stackable-base.

Reply:
I've not used config-utils before: not sure how to template the file without first declaring the env-vars. I'll have a look.

Reply:
Will defer this to https://github.com/stackabletech/docker-images/issues/1191